tests/test_flows.sh

```diff
@@ -509,7 +509,7 @@ test_client_steering_dummy() {
     send_bwl_event ${REPEATER1} wlan2 "EVENT AP-STA-CONNECTED ${sta_mac}"
     dbg "Confirm steering success by client connected"
     check_log ${GATEWAY} controller "steering successful for sta ${sta_mac}"
-    check_log ${GATEWAY} controller "sta ${sta_mac} disconnected after successful steering"
+    check_log ${GATEWAY} controller "sta ${sta_mac} disconnected after steering, proceeding to unblock"
```

Comment:
Please squash this with the commit that changes the message. And perhaps do that in a separate commit, not together with the change of what "success" means.